L1Trigger/VertexFinder/python/VertexProducer_cff.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
```

Comment:
This file should be a `cfi` (it defines 1 module), not a `cff`.

Reply:
Done